src/evict/evict_lru.c

```diff
@@ -1216,6 +1216,10 @@ __evict_walk_file(WT_SESSION_IMPL *session, u_int *slotp)
 		page = ref->page;
 		modified = __wt_page_is_modified(page);
 
+		/* Don't add clean pages if we are running in-memory */
+		if (F_ISSET(S2C(session), WT_CONN_IN_MEMORY) && !modified)
+			continue;
+
```

Comment:
@agorrod, I'd move this down about 10 lines and merge with the existing test for skipping clean pages, doesn't that allow us to get rid of empty pages, which is a good thing?